### PR TITLE
feat: SingleProject - ToDo

### PR DESCRIPTION
**Pull request by vatsashah45**

### CODE REVIEW

File: reference/ToDo/ToDo/ToDo/ToDo.csproj [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      $(DotNetVersion)-browserwasm;
+      $(DotNetVersion)-android;
+      $(DotNetVersion)-ios;
+      $(DotNetVersion)-maccatalyst;
+      $(DotNetVersion)-windows10.0.19041;
+      $(DotNetVersion)-desktop;
+    </TargetFrameworks>
+
+    <OutputType>Exe</OutputType>
+    <SingleProject>true</SingleProject>
```
Comment (eriklimakc) [inline reference/ToDo/ToDo/ToDo/ToDo.csproj:13]:
```suggestion
    <UnoSingleProject>true</UnoSingleProject>
```

File: reference/ToDo/ToDo/Directory.Build.props [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      PRI257: Ignore default language (en) not being one of the included resources (eg en-us, en-uk)
+    -->
+    <NoWarn>$(NoWarn);NU1507;NETSDK1201;PRI257</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <UnoExtensionsVersion>4.1.0-dev.74</UnoExtensionsVersion>
+    <UnoToolkitVersion>5.2.0-dev.51</UnoToolkitVersion>
+    <UnoThemesVersion>4.2.0-dev.26</UnoThemesVersion>
+  </PropertyGroup>
```
Comment (eriklimakc) [inline reference/ToDo/ToDo/Directory.Build.props:21]: Would be nice to update those to a more up-to-date version.

**Review by eriklimakc — CHANGES_REQUESTED** [review]

**Comment by eriklimakc** [comment]
## Known issues

- [ ] Resizing the app to make it narrow on **Windows** causes the app to crash
- [ ] When trying to launch on **Desktop** an exception is thrown: `Could not load file or assembly 'Uno.Extensions.Authentication.MSAL.WinUI"`.
- [ ] Data not being loaded on **Android**. "An error occured." message is displayed. Suspecting of https://github.com/unoplatform/uno/issues/16430

**Comment by agneszitte** [comment]
@eriklimakc, @vatsashah45 
@dansiegel left some details for https://github.com/unoplatform/uno/issues/16439#issuecomment-2083258997
Are there any additional blockers left for migrating ToDo, please?
(cc @nickrandolph)

**Comment by agneszitte** [comment]
fyi @eriklimakc I currently have some local changes with the latest versions and additional fixes.
I will finish the work tomorrow and list the leftover issues

**Comment by agneszitte** [comment]
Closing this PR in favor of https://github.com/unoplatform/Uno.Samples/pull/745

**Pull request closed without merging**